birdhouse/components/cowbird/config/cowbird/config.yml.template

```diff
@@ -14,8 +14,7 @@ handlers:
     admin_password: ${MAGPIE_ADMIN_PASSWORD}
   Geoserver:
     active: true
-    # FIXME: replace by the protected/proxy endpoint once component is made available
-    url: http://${BIRDHOUSE_FQDN_PUBLIC}:8765/geoserver
+    url: http://${BIRDHOUSE_FQDN_PUBLIC}/geoserver
```

Comment:
Shouldn't that be either the internal access (`http://geoserver:????`) or using HTTPS?
A mix of `http` and `${BIRDHOUSE_FQDN_PUBLIC}` seems incorrect.

Ideally, it would use the internal connection to avoid the unnecessary loop-back.

Reply:
So I thought so too but the above comment is `"replace by the protected/proxy endpoint once component is made available"` which says to me that someone intentionally wanted this to be the public URL. 

I'm happy to change it to the `http://geoserver:...` version since I don't see why we have to use the public one really.

Reply:
Yes, that's an old comment from myself related to the removal of protected ports. I think at that time GeoServer was still accessed by HTTP:PORT so it couldn't be updated. Most probably not applicable anymore. 

Since Cowbird acts "behind the scene" with admin-only access specific to GeoServer, I think it's safe to skip the extra Magpie/Twitcher layer.